README.md

```diff
@@ -1,156 +1,202 @@
-# Hemi Network
+# ⚡️ The Hemi Network
+
+Hemi is an EVM-compatible L2 blockchain that combines the security of Bitcoin with the programmability of Ethereum.
 
 <details>
   <summary>Table of Contents</summary>
 
-* [What is the Hemi Network?](#what-is-the-hemi-network)
-  * [Services](#services)
-  * [License](#license)
-* [Getting Started](#getting-started)
-  * [Building from Source](#building-from-source)
-  * [Downloading Binaries from Releases](#downloading-binaries-from-releases)
-  * [Running](#running)
-  * [Running popmd](#running-popmd)
+<!-- TOC -->
+* [⚡️ The Hemi Network](#-the-hemi-network)
+  * [🔧 Services](#-services)
+  * [🌐 Binaries](#-binaries)
+* [⚡️ Getting Started](#-getting-started)
+  * [📦 Downloading Release Binaries](#-downloading-release-binaries)
+  * [🏗 Building from Source](#-building-from-source)
+    * [🏁 Prerequisites](#-prerequisites)
+    * [Building with Makefile](#building-with-makefile)
+* [🛠 Running the Services](#-running-the-services)
+  * [▶️ Running popmd](#-running-popmd)
+    * [🏁 Prerequisites](#-prerequisites-1)
     * [CLI](#cli)
     * [Web](#web)
-  * [Running bfgd](#running-bfgd)
-  * [Running bssd](#running-bssd)
-
+  * [▶️ Running bfgd](#-running-bfgd)
+    * [🏁 Prerequisites](#-prerequisites-2)
+  * [▶️ Running bssd](#-running-bssd)
+    * [🏁 Prerequisites](#-prerequisites-3)
+  * [▶️ Running the localnet network](#-running-the-localnet-network)
+    * [🏁 Prerequisites](#-prerequisites-4)
+    * [📚 Tutorial](#-tutorial)
+  * [📄 License](#-license)
+<!-- TOC -->
 </details>
 
-## What is the Hemi Network?
+---
 
-Hemi is an EVM compatible L2 blockchain that brings Bitcoin security and Ethereum programability together.
+## 🔧 Services
 
-### Services
+The Hemi Network comprises three key services, each serving a unique function within the network:
```

Comment:
Is that grammatically reasonable?  I'd go with `The Hemi network is compriseed of three key services` or something more like that.

Reply:
"is comprised of" seems to be less correct, I have reworded it to use "consists of" - is that okay?

Reply:
Yes, better than mine.